test/python/test_axpy_pipelined.com

```diff
@@ -0,0 +1,33 @@
+import sys
+import pytest
+import getpass
+import tensorflow as tf
+import ngraph_bridge
+
+import numpy as np
+from common import NgraphTest
+
+# This test needs access to mnist_deep_simplified.py script
+# present in ngraph-bridge/examples/mnist
+# so this path needs to be added to python path when running this
+
+# For eg. when running the test from ngraph-bridge/build_cmake/test/python
+# you can add this path as below
+#sys.path.insert(0, '../../examples')
+
+from axpy_pipelined import *
+
+
+class TestAxpyPipelined(NgraphTest):
+
+    def test_axpy_pipelined(self):
+        input_array, output_array, expected_output_array = run_axpy_pipeline()
+        for i in range(1, 10):
```

Comment:
```suggestion
        for i in range(10):
```

then the `[i-1]` becomes `[i]` in `output_array` and `expected_output_array`

Reply:
Ah, I see the `range(1,10)` carried over from the axpy example.

Reply:
I going to keep it as it is.